content/concepts/content-routing/overview.md

```diff
@@ -4,4 +4,56 @@ description: "Learn about Noise in libp2p."
 weight: 218
 ---
 
-Coming soon!
+## Overview
+
+Content routing refers to the process of directing data to its intended recipient within a
+P2P network. In a traditional client-server network, routing is a straightforward process
+because a central authority manages the flow of data. However, in a P2P network, there is
+no central authority, so routing becomes more complex.
+
+One of the main challenges of content routing in P2P networks is scalability. As the number
+of peers in a network increases, the number of possible routes for data to travel also increases.
+This can lead to a situation where the routing table becomes excessively large and difficult to
+manage.
+
+Another challenge is fault tolerance. In a centralized network, if the central server goes down,
```

Comment:
While true. This is also true of anything p2p. Not sure why this should show up in content routing. I see content routing as just the interface that other things use. Kademlia is one implementation of that interface.